portable/ThirdParty/GCC/Posix/portmacro.h

```diff
@@ -70,7 +69,7 @@ typedef unsigned long TickType_t;
 #define portSTACK_GROWTH			( -1 )
 #define portHAS_STACK_OVERFLOW_CHECKING	( 1 )
 #define portTICK_PERIOD_MS			( ( TickType_t ) 1000 / configTICK_RATE_HZ )
-#define portTICK_RATE_MICROSECONDS	( ( portTickType ) 1000000 / configTICK_RATE_HZ )
+#define portTICK_RATE_MICROSECONDS	( ( TickType_t ) 1000000 / configTICK_RATE_HZ )
```

Comment:
What are the implications of removing `portTickType` here?

`TickType_t` is limited to 16 or 32 bits and one utility of having `portTickType` here is that 64-bit posix targets can define `portTickType` as 64-bit instead, allowing finer granularity.


I'm not very familiar with how our Posix handles/quantifies tick cycles.
However I can say that

Reply:
Is there specific issue that removing `portTickType` here solves? Why the removal?

Reply:
There are no specific issues, I just wanted to fix a compilation error.